mongo/bboard_v1.json

```diff
@@ -0,0 +1,76 @@
+[
+    {
+        "//": "Comment: bountyHash message -> bankless dao bounty board"
+    },
+    {
+        "season" : 2,
+        "bountyTitle": "A new About Us webpage",
+        "bountyDescription": "Description of webpage",
+        "doneCriteria": "A completed bounty has the following...",
+        "bountyReward": {
+            "currency" : "BANK",
+            "amount" : 5000
+        },
+        "applicableGuilds": [
+            {"guildName": "Marketing Guild"},
+            {"guildName": "Treasury Guild"},
+            {"guildName": "Developer's Guild"},
+            {"guildName": "Analytics Guild"},
+            {"guildName": "Writer's Guild"}
+        ],
+        "bountyCreatedBy": {
+            "isDaoMember" : true,
+            "guildName" : "Analytics Guild",
+            "discordHandle" : "@paulapivat#8888",
+            "publicAddress" : "0x2d94aa3e47d9d5024503ca8491fce9a2fb4da198"
```

Comment:
Including the wallet address will be great for MVP 2 and future iterations, for now I think we could leave this out as bounties will be distributed through the social layer for MVP 1.

Reply:
Ok, will take out "publicAddress" field for now.